CDTS Blazor/Pages/IdentityDocuments.razor

```diff
@@ -32,17 +32,75 @@
         </div>
     </form>
 
+    @if (@loading == true)
+    {
+        // TODO : Add a loading animation instead of just text.
```

Comment:
```suggestion
        // TODO : Add a loading animation instead of just text.
        // Aurora: https://design.gccollab.ca/component/icons
        // WET: https://wet-boew.github.io/wet-boew-styleguide/design/icons-en.html
        // Bootstrap: https://www.blazorade.com/Blazorade-Bootstrap/spinners
```